src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,282 @@
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Description      | RECOMMENDED. Free-form natural language description of the nature of the file.                                                                                                                                                                                                                                                                        |
+| Sources          | OPTIONAL. A list of paths relative to dataset root pointing to the file(s) that were directly used in the creation of this derivative. For example in a chain of A->B->C, “C” should only list “B” as Sources, and “B” should only list “A” as Sources. However in case X and Y jointly contribute to Z, then “Z” should list “X” and “Y” as Sources. |
+| RawSources       | OPTIONAL. A list of paths relative to dataset root pointing to the BIDS-Raw file(s) that were used in the creation of this derivative. When the derivative filename does not define a `space` keyword, the first entry of `RawSources` MUST be defined and it will define the `scanner` coordinate system that applies.                               |
+| SpatialReference | REQUIRED when a custom template or file is used. A path to a file that was used as, or can be used as, a reference image for determining the coordinate space of this file. The path should start with a `/` and should be relative to the root of the dataset.                                                                                       |
+
+### SpatialReference key allowed values
+
+| **Value name** | **Description**                                                                                                               |
+| -------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| orig           | A (potentially unique) per-image space. Useful for describing the source of transforms from an input image to a target space. |
+| uri or path    | This can be used to point to a specific file.                                                                                 |
+
+### Examples
+
+Preprocessed `bold` NIfTI file in `individual` coordinate space. Please mind
+that in this case `SpatialReference` key is REQUIRED.
+
+```Text
+sub-01/func/sub-01_task-rest_space-individual_bold.nii.gz
+sub-01/func/sub-01_task-rest_space-individual_bold.json
+```
+
+```JSON
+{
+    "SpatialReference": "/sub-01/anat/sub-01_desc-combined_T1w.nii.gz"
+}
+```
+
+Participant cortical thickness GIFTI file in `individual` coordinate space.
```

Comment:
`_thickness` is used in this PR but never really introduced as an officially supported suffix.  May be it is ok, and would just need to wait for #207 which [introduces it](https://github.com/bids-standard/bids-specification/pull/207/files#diff-4ff87c6142082e27a0f84319b01b85ffR89) and which I guess will chase this PR?

Reply:
Let's defer this example to come along with the `_thickness` suffix.